### PR TITLE
Add support for showing "footsteps" on the map

### PR DESCRIPTION
**Pull request by evanw555**
Adds a config which allows the user to specify the length (in tiles) of
a trail to be rendered on the world map showing where some GIMP has
walked. The color of the trail depends on the index of the GIMP in his
group (the colors for now are hardcoded).

Also changes the smooth map point motion logic to smoothly move even
between different planes. I just realized that "planes" seem to mostly
refer to different stories/levels within the same area. Moving to a
completely different area (e.g. ZMI) doesn't seem to affect the plane.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,8 +37,17 @@ dependencies {
 
 group = 'com.gimp'
 version = '2.0'
-sourceCompatibility = '1.8'
+sourceCompatibility = isGimpDev ? '11' : '8';
```
Comment (davidvorona) [inline build.gradle:40]: does 8 == 1.8?

Comment (evanw555) [reply build.gradle:40]: Ohhhh that should be `1.8`, let me fix that (but yeah technically it's the same thing)

File: build.gradle [hunk 2]
```diff
@@ -12,8 +13,15 @@ repositories {
 
 def runeLiteVersion = '1.8.9'
 
+// Test locally via Gradle: "./gradlew run -DgimpDev=true"
```
Comment (davidvorona) [inline build.gradle:16]: `./gradlew run -DgimpDev=true` is this supposed to have space => `./gradlew run -D gimpDev=true`?

Comment (evanw555) [reply build.gradle:16]: Nope. For some reason, Gradle System properties (`-Dsomething`) and project properties (`-Psomething`) don't have spaces between the CLI flag and the value. It's weird, but that's how it is.

**Review by davidvorona — APPROVED** [review]

**Pull request merged**